test/trace.spec.ts

```diff
@@ -111,3 +113,19 @@ it('should record trace with POST', (test, { browserName, platform }) => {
   expect(fs.existsSync(path.join(traceDir, 'resources', resourceEvent.requestSha1))).toBe(true);
   expect(fs.existsSync(path.join(traceDir, 'resources', resourceEvent.responseSha1))).toBe(true);
 });
+
+it('should record trace with a debugName', (test, { browserName, platform }) => {
+  test.fixme();
```

Comment:
Tracing tests are flaky on the bots, I still need to figure out why and reenable them.

Reply:
What do you mean by bots?

Reply:
I mean our GitHub actions.

Reply:
Ah ok thanks